### PR TITLE
Add method to store CIS data to DynamoDB.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

File: cis/validation.py [hunk 1]
```diff
@@ -42,3 +44,24 @@ def validate(**payload):
         return False
 
     return True
+
+
+def store_to_vault(data):
```
Comment (akatsoulas) [inline cis/validation.py]: Shall we add a check for the data format somewhere in this function?

File: tests/test_validation.py [hunk 2]
```diff
@@ -87,3 +88,63 @@ def test_invalid_kms_key(self, mock_kms):
...
+
+        dynamodb_mock.Table.return_value = dynamodb_table
+        mock_boto.resource.return_value = dynamodb_mock
+
+        data = {
+            'foo': 'bar',
+            'foobar': 42
+        }
+
+        from cis.validation import store_to_vault
```
Comment (akatsoulas) [inline tests/test_validation.py]: Is this causing circular dependencies? If not, let's move it to the top.

File: tests/test_validation.py [hunk 3]
```diff
@@ -87,3 +88,63 @@ def test_invalid_kms_key(self, mock_kms):
...
+        dynamodb_table.put_item.side_effect = Exception('DynamoDB exception')
+        dynamodb_mock.Table.return_value = dynamodb_table
+        mock_boto.resource.return_value = dynamodb_mock
+
+        data = {
+            'foo': 'bar',
+            'foobar': 42
+        }
+
+        from cis.validation import store_to_vault
```
Comment (akatsoulas) [inline tests/test_validation.py]: Same here.

File: tests/test_validation.py [hunk 4]
```diff
@@ -87,3 +88,63 @@ def test_invalid_kms_key(self, mock_kms):
             from cis.validation import validate
             is_valid_payload = validate(**payload)
         self.assertFalse(is_valid_payload)
+
+
+class DatabaseTest(unittest.TestCase):
```
Comment (akatsoulas) [inline tests/test_validation.py]: Shall we add another test with malformed data?

File: cis/validation.py (hunk 1 above)
Comment (johngian) [inline cis/validation.py]: No let's keep it atomic. The validation is happening in a different level.

File: tests/test_validation.py (hunk 4 above)
Comment (johngian) [inline tests/test_validation.py]: Validation is tested in the TestCase above.

File: tests/test_validation.py (hunk 2 above)
Comment (johngian) [inline tests/test_validation.py]: We need this here for setUp to work correctly (we are patching env vars) in all our testcases

File: tests/test_validation.py (hunk 3 above)
Comment (johngian) [inline tests/test_validation.py]: Same as above.

**Pull request merged**